### PR TITLE
Fix time limitation in magma_free docstring

### PR DESCRIPTION
**Pull request by Eloitor**
Change time limitation from 20 to 120 in the documentation of magma_free.

> Calculations are restricted to 120 seconds.
http://magma.maths.usyd.edu.au/calc/

> <max_time>120</max_time>
http://magma.maths.usyd.edu.au/xml/calculator.xml?input=Factorization(15)

### :memo: Checklist

- [x] The title is concise and informative.
- [x] The description explains in detail what this PR is about.
- [ ] I have linked a relevant issue or discussion.
- [ ] I have created tests covering the changes.
- [ ] I have updated the documentation and checked the documentation preview.

### CODE REVIEW

**Comment by vincentmacri** [comment]
@Eloitor since the CI is failing to build the docs, before I approve this I want to ask if you've built the documentation locally?

**Comment by Eloitor** [comment]
I didn't for this small change.

File: src/sage/interfaces/magma_free.py
Comment (vincentmacri) [inline src/sage/interfaces/magma_free.py]:
```suggestion
    .. WARNING::
    
        The code must evaluate in at most 120 seconds
```

Comment (vincentmacri) [reply src/sage/interfaces/magma_free.py]: I think this is the proper way to warn the user of something like this: https://doc.sagemath.org/html/en/developer/coding_basics.html#the-docstring-of-a-function-content

(Also pushing another commit will re-run the CI and hopefully the error will go away)

File: src/sage/interfaces/magma_free.py
Comment (vincentmacri) [inline src/sage/interfaces/magma_free.py]:
```suggestion
        and there is a limitation on the amount of RAM.
```

This should make the spacing match up with the other suggestion

File: src/sage/interfaces/magma_free.py
Comment (tscrim) [inline src/sage/interfaces/magma_free.py]:
```suggestion

```
Please remove the trailing whitespace.

**Review by tscrim — CHANGES_REQUESTED** [review]

File: src/sage/interfaces/magma_free.py
Comment (vincentmacri) [reply src/sage/interfaces/magma_free.py]: I think a blank line between the block header and the block content is part of our style, no? It's in all the examples here: https://doc.sagemath.org/html/en/developer/coding_basics.html#the-docstring-of-a-function-content

File: src/sage/interfaces/magma_free.py
Comment (vincentmacri) [inline src/sage/interfaces/magma_free.py:29]:
```suggestion
    .. WARNING::

```

I think @tscrim's suggestion was to remove the spaces, not the blank line.

**Comment by vincentmacri** [comment]
LGTM

File: src/sage/interfaces/magma_free.py
Comment (tscrim) [reply src/sage/interfaces/magma_free.py:29]: Yes, that’s right. GH seems to have thought I meant to remove the entire line…but that’s not what was intended.

**Review by tscrim — APPROVED** [review]

**Pull request merged**
